doc/ring/static.md

```diff
@@ -0,0 +1,89 @@
+# Static Resources (Clojure Only)
+
+Static resources can be served with a help of `reitit.ring/create-resource-handler`. It takes optionally an options map and returns a ring handler to serve files from Classpath. It returns `java.io.File` instances, so ring adapters can use NIO to effective Stream the files.
+
+There are two options to serve the files.
+
+## Internal routes
+
+This is good option if static files can be from non-conflicting paths, e.g. `"/assets/*"`.
+
+
+```clj
+(require '[reitit.ring :as ring])
+
+(ring/ring-handler
+  (ring/router
+    [["/ping" (constantly {:status 200, :body "pong"})]
+     ["/assets/*" (ring/create-resource-handler)]])
+  (ring/create-default-handler))
+```
+
+To serve static files with conflicting routes, e.g. `"/*#`, one needs to disable the confligt resolution:
+
+```clj
+(require '[reitit.ring :as ring])
+
+(ring/ring-handler
+  (ring/router
+    [["/ping" (constantly {:status 200, :body "pong"})]
+     ["/*" (ring/create-resource-handler)]]
+    {:conflicts (constantly nil)})
+  (ring/create-default-handler))
+```
+
+## External routes
+
+To serve files from conflicting paths, e.g. `"/*"`, one option is to mount them to default-handler branch of `ring-handler`. This way, they are only served if none of the actual routes have matched.
+
+
+```clj
+(ring/ring-handler
+  (ring/router
+    ["/ping" (constantly {:status 200, :body "pong"})])
+  (ring/routes
+    (ring/create-resource-handler {:path "/"})
+    (ring/create-default-handler)))
+```
+
+## Configuration
+
+`reitit.ring/create-resource-handler` takes optionally an options map to configure how the files are being served.
+
+| key          | description |
+| -------------|-------------|
+| :parameter   | optional name of the wildcard parameter, defaults to unnamed keyword `:`
+| :root        | optional resource root, defaults to `"public"`
+| :mime-types  | optional extension->mime-type mapping, defaults to `reitit.ring.mime/default-types`
+| :path        | optional path to mount the handler to. Works only if mounted outside of a router.
+
+### TODO
+
+* support for things like `:cache`, `:last-modified?` and `:index-files`
+
+## Performance
```

Comment:
It's a bit unclear to me if this perf section is useful. There's no point of comparison for the numbers below and anyway a single number isn't very informative – usually you'd want to look at the distribution.

Besides, I tried running the large file command against my web app that uses Aleph, `ring.util.response/resource-response` and custom regexp-based routing on my mid-2015 MBP and got this:

```
Requests/sec:   6398.48
Transfer/sec:      2.48GB
```

I didn't try reitit on my computer, but just based on the text here I would've expected this benchmark to show higher numbers than what I'd get with my setup.

If you'd like to include a point of comparison, I think something like nginx might be a good one. It's the de facto server for serving static files, after all.